projects/kit/components/checkbox/checkbox-options.ts

```diff
@@ -0,0 +1,19 @@
+import {InjectionToken} from '@angular/core';
+import {TuiSizeL} from '@taiga-ui/core';
+
+export interface CheckboxOptions {
+    size: TuiSizeL;
+}
+
+/** Default values for the checkbox options. */
+export const TUI_CHECKBOX_DEFAULT_OPTIONS: CheckboxOptions = {
```

Comment:
This one is also `Readonly<CheckboxOptions>` so nobody decides to mutate the constant instead of providing :) We love `readonly` here :)